pf/tfbridge/provider.go

```diff
@@ -243,37 +243,3 @@ func newProviderServer6(ctx context.Context, p pfprovider.Provider) (tfprotov6.P
 
 	return server6, nil
 }
-
-type packageSpec struct {
```

Comment:
This just moves around to facilitate testing. Can also remove. A bit of premature abstraction perhaps.